lib/manager/poetry/artifacts.spec.ts

```diff
@@ -101,6 +103,7 @@ describe('.updateArtifacts()', () => {
         newPackageFileContent: '{}',
         config: {
           ...config,
+          compatibility: { python: '3.7' },
```

Comment:
Maybe we should have two docker tests - one for the autodetection of python version and another using the configured constraint.